### PR TITLE
nvidia-trt: no stop request, stop response for decoupled model

### PR DESCRIPTION
**Pull request by mkhludnev**
- **Description:**  
    - suppress explicit stop request since Triton doesn't support it. https://github.com/triton-inference-server/server/issues/4818
    - detect decoupled model and ask for explicit stop response 
 
- **Issue:** now it sends `stop:True` as an input, and triton rejects it; then without `triton_enable_empty_final_response=True` stream hangs forever 

- [ ] **Add tests and docs**: If you're adding a new integration, please include
  1. a test for the integration, preferably unit tests that do not rely on network access,
  2. an example notebook showing its use. It lives in `docs/docs/integrations` directory.

- [ ] **Lint and test**: Run `make format`, `make lint` and `make test` from the root of the package(s) you've modified. See contribution guidelines for more: https://python.langchain.com/docs/contributing/

If no one reviews your PR within a few days, please @-mention one of baskaryan, efriis, eyurtsev, hwchase17.

### CODE REVIEW

File: libs/partners/nvidia-trt/langchain_nvidia_trt/llms.py [hunk 1]
```diff
@@ -218,7 +218,7 @@ def _invoke_triton(self, model_name, inputs, outputs, stop_words):
         result_queue = StreamingResponseGenerator(
             self,
             request_id,
-            force_batch=False,
+            signal_stop=False,
```
Comment (mkhludnev) [inline libs/partners/nvidia-trt/langchain_nvidia_trt/llms.py:221]: renaming to meaningful. Inverting it at line 405. 
TLDR triton doesn't let one cancel a request https://github.com/triton-inference-server/server/issues/4818

File: libs/partners/nvidia-trt/langchain_nvidia_trt/llms.py [hunk 2]
```diff
@@ -238,6 +238,7 @@ def _invoke_triton(self, model_name, inputs, outputs, stop_words):
             inputs=inputs,
             outputs=outputs,
             request_id=request_id,
+            enable_empty_final_response=self.client.get_model_config(model_name).config.model_transaction_policy.decoupled
```
Comment (mkhludnev) [inline libs/partners/nvidia-trt/langchain_nvidia_trt/llms.py:241]: I want to improve it here. Maybe cache it or use it as model_ready call above. I also don't know how much this grpc code is null proof.

**Comment by efriis** [comment]
Should be against the https://github.com/langchain-ai/langchain-nvidia repo instead!
